src/main/asciidoc/readme.adoc

```diff
@@ -172,8 +172,9 @@ The jQAssistant Maven plugin must be configured in the pom.xml of the root modul
                         ${project.build.directory}/jqassistant/jqassistant-report.xml
                     </xmlReportFile>
                 </configuration>
+                -->
+                <!-- additional plugins -->
                 <dependencies>
```

Comment:
This comment section illustratries the possible options to configure the Maven plugin and their defaults. For that reason the additional plugin was also part of the comment (it's not added as dependency by default). Recommending to use a separte comment block for this